packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -377,6 +377,7 @@ function ShellHeader() {
             <div className="mb-1 h-6 w-32 animate-pulse rounded-md bg-gray-200" />
           )}
         </div>
+        <div className="ml-auto">{meta.CTA}</div>
```

Comment:
Expand this section and you will see `ShellHeader` This is where the `useMeta()` hook is called and returns `MetaContext`. By adding the CTA to the `ShellHeader`, the CTA will not rerender with the rest of the page when changes are made.